shotover-proxy/src/transforms/cassandra/sink_cluster/node_pool.rs

```diff
@@ -131,19 +145,37 @@ impl NodePool {
                 .clone()
         };
 
+        let keyspace = {
```

Comment:
Need to handle this error better, we should just do normal routing if we don't have the keyspace info.